Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Shared/Crashes.cs

```diff
@@ -197,6 +197,45 @@ public static GetErrorAttachmentsHandler GetErrorAttachments
             }
         }
 
+#if ENABLE_IL2CPP
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+#endif
+        public delegate void SendingErrorReportHandler(ErrorReport errorReport);
+
+        public static SendingErrorReportHandler SendingErrorReport
```

Comment:
Please convert properties to events according to documentation:
https://docs.microsoft.com/en-us/appcenter/sdk/crashes/unity#the-following-callback-will-be-invoked-before-the-sdk-sends-a-crash-log